public/hyperswitch/index.html

```diff
@@ -1,108 +1,132 @@
-    <meta charset="UTF-8" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600;700&family=IBM+Plex+Sans:wght@300;400;500;600;700&display=swap"
-      rel="stylesheet"
-    />
-    <style>
-      html,
-      body {
-        font-family: "Inter", sans-serif;
-        font-weight: 400;
-        font-size: 16px;
-        -webkit-font-smoothing: antialiased;
-        padding: 0;
-        margin: 0;
-        height: 100%;
-        width: 100%;
-        display: flex;
-        flex-direction: column;
-      }
 
-      #app {
-        display: flex;
-        flex-direction: column;
-        flex: 1;
-      }
+<head>
+  <title>Loading...</title>
 
-      .list-item {
-        background-color: rgba(255, 255, 255, 1);
-        padding: 10px;
-        transition: background-color 0.2s;
-      }
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta charset="UTF-8" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preload"
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+    as="style" />
+  <link rel="preload"
+    href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600;700&family=IBM+Plex+Sans:wght@300;400;500;600;700&display=swap"
+    as="style" />
 
-      .list-item:hover {
-        background-color: rgba(0, 0, 0, 0.1);
-      }
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+    rel="stylesheet" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600;700&family=IBM+Plex+Sans:wght@300;400;500;600;700&display=swap"
+    rel="stylesheet" />
 
-      .clickable {
-        cursor: pointer;
-      }
+  <style>
+    @font-face {
+      font-family: 'Inter';
+      src: url('https://fonts.gstatic.com/s/inter/v11/UcCo3FwrMFzOb18.woff2') format('woff2');
+      font-display: swap;
+    }
 
-      .react-pdf__Page__annotations {
-        height: auto !important;
-        width: 0 !important;
-      }
+    @font-face {
+      font-family: 'IBM Plex Mono';
+      src: url('https://fonts.gstatic.com/s/ibmplexmono/v6/zYXkKVElMYYaJe8bpLHnCwDKhd.woff2') format('woff2');
+      font-display: swap;
+    }
 
-      .linkAnnotation {
-        width: 20% !important;
-      }
+    @font-face {
+      font-family: 'IBM Plex Sans';
+      src: url('https://fonts.gstatic.com/s/ibmplexsans/v8/zYXgKVElMYYaJe8bpLHnCwCzB.woff2') format('woff2');
+      font-display: swap;
+    }
```

Comment:
I dont think we need this as we are passing in the href of those google fonts

Reply:
@JeevaRamu0104 can we remove one of the font as I guess we are only using the Inter or IBM only one. Why to fetch 2 fonts unnecessarily.